File: 프로그래머스/unrated/181896. 첫 번째로 나오는 음수/첫 번째로 나오는 음수.py
def solution(num_list):
    # 방법 2 - enumerate 함수
    for idx, el in enumerate(num_list):
        if el < 0: return idx
    return -1

    # # 방법 1 - for문
    # for i in range(len(num_list)):
    #     if num_list[i] < 0: return i
    # return -1

File: 프로그래머스/unrated/181896. 첫 번째로 나오는 음수/첫 번째로 나오는 음수.js
function solution(num_list) {
    // 방법 2 - findIndex 함수
    return num_list.findIndex(el => el < 0)
    
    // // 방법 1 - for문
    // for (let i = 0; i < num_list.length; i++) {
    //     if (num_list[i] < 0) return i
    // }
    // return -1
}